src/harness/reference_models/common/data.py

```diff
@@ -265,13 +265,16 @@ def getAuthorizedGrantsFromDomainProxies(domain_proxies):
   Returns:
     A list of |CbsdGrantInfo| for each authorized grant in the given Domain Proxies.
   """
-  grants = []
+  grant_records = []
+  cbsd_records = []
   for domain_proxy in domain_proxies:
     for cbsd in domain_proxy.getCbsdsWithAtLeastOneAuthorizedGrant():
+      reg_record = cbsd.getRegistrationRequest()
       for grant in cbsd.getAuthorizedGrants():
-        grants.append(
-            constructCbsdGrantInfo(
-                cbsd.getRegistrationRequest(),
-                grant.getGrantRequest()))
-
-  return grants
+        grant_record = grant.getGrantRequest()
```

Comment:
Will this work correctly? I assume that you're trying to construct the `GrantData` object from SAS-SAS, and it has a slightly different format than the `GrantRequest` object. Where is the output of this function used?

It also looks like the output type of this function has changed; will the code that uses this function still work?

Reply:
Taran has modified Aggregate interference code in PR621, so this changes are not required now, Will remove.